### PR TITLE
feat(#13): stopped using git

### PR DESCRIPTION
**Pull request by oxmanroman**

### CODE REVIEW

**Comment by oxmanroman** [comment]
Ensure to merge https://github.com/lightning-resume/lightning-resume/pull/12 before this

File: src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * The issue here is that the zip contains a root folder with the name of the repo
+ * So the files will actually be inside ~/my/path/myRepo
+ *
+ * 4. move all the files up from ~/my/path/myRepo to ~/my/path
+ *
+ * @param url repository url
+ * @param outputPath output path
+ * @returns
+ */
+export async function downloadRepository(url: string, outputPath: string): Promise<void> {
```
Comment (chekuda) [inline src/utils.ts:26]: outputPath is the path where to copy the files no?

File: src/utils.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const files = fs.readdirSync(deepPath);
+  await Promise.all(files.map((file) => fs.move(path.join(deepPath, file), path.join(outputPath, file))));
+  fs.removeSync(deepPath);
+}
+
+export function httpDownload(url: string, outputPath: string): Promise<void> {
+  return new Promise((resolve, reject) => {
+    fs.ensureDirSync(path.join(outputPath, '..'));
+
+    const download = wget.download(url, outputPath);
```
Comment (chekuda) [inline src/utils.ts:46]: Maybe not bad idea to promisify all!! https://github.com/ajhsu/node-wget-promise

**Review by chekuda — APPROVED** [review]
Good one mate

File: src/utils.ts (hunk 1 above)
Comment (oxmanroman) [reply src/utils.ts:26]: yep yep

File: src/utils.ts (hunk 2 above)
Comment (oxmanroman) [reply src/utils.ts:46]: that looks nice! the thing is that we are using the `wget-improved`...
Usually, the sync methods of fs kinda makes things simpler for us in this case

Comment (chekuda) [reply src/utils.ts:46]: OK, just thought that u want to use promises.

**Pull request merged**